Core/Core/Logging/SpeckleLog.cs

```diff
@@ -236,11 +241,27 @@ SpeckleLogConfiguration logConfiguration
     }
 
     var logger = serilogLogConfiguration.CreateLogger();
+
+    // Configure scope after logger created.
+    SentrySdk.ConfigureScope(scope =>
+    {
+      scope.User = new User { Id = id };
+    });
+    SentrySdk.ConfigureScope(scope =>
+    {
+      scope.SetTag("hostApplication", hostApplicationName);
+    });
```

Comment:
I'm assuming this scope is global as it has no reference to the logger instance.

This would collide with anybody else using Sentry on the same AppDomain

Reply:
When I remove this, nothing really effected on logs

Reply:
I have to update this comment, because I was checking it on Seq. It should have meaningful impact on sentry.